src/officehours_ui/templates/base.html

```diff
@@ -47,45 +54,60 @@
-                    Manage Queues
-                  </a>
-                  <a class="dropdown-item" href="{% url 'preferences' %}" style="background-color: white; color:#212529">
-                    Preferences
-                  </a>
-                  <form action="{% url 'oidc_logout' %}" method="post">
-                    {% csrf_token %}
-                    <input type="submit" value="Logout" style="background-color: white" class="dropdown-item">
-                  </form>
-                </div>
-              </li>
+                <form action="{% url 'oidc_logout' %}" method="post">
+                  {% csrf_token %}
+                  <input type="submit" value="Logout" style="background-color: white" class="dropdown-item">
+                </form>
+              </div>
+            </li>
             {% else %}
-              <a href="{{ LOGIN_URL }}" class="nav-link">Login</a>
+            <a href="{{ LOGIN_URL }}" class="nav-link">Login</a>
             {% endif %}
           </div>
         </div>
     </nav>
   </header>
-  
+
   {% for page in flatpages %}
-    {% if page.title == "User Comm Banner" %}
-      {{page.content|safe}}
-    {% endif %}
+  {% if page.title == "User Comm Banner" %}
+  {{page.content|safe}}
+  {% endif %}
   {% endfor %}
 
-	<div id="main-content" class="container" role="main">
+  <div id="main-content" class="container" role="main">
     {% block content %}{% endblock %}
     {% csrf_token %}
-	</div>
+  </div>
+
+  <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
+    integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
+    crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"
+    integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1"
+    crossorigin="anonymous"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"
+    integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
+    crossorigin="anonymous"></script>
 
-  <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
-  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+
+  <script>
+    $('#navbar-toggle').click(function () {
+      $(this).find('i').toggleClass('fas fa-bars text-light').toggleClass('fas fa-times');
+    });
+  </script>
```

Comment:
This approach will work. However -- and maybe this is too much futureproofing -- I know that Bootstrap 5 (we're on 4) drops JQuery as a dependency, and I don't believe we're explicitly using it anywhere else in the project like this.

I think I'd like to see something like this instead, just to prevent future migration/dependency work.
```
    const navBarToggleIconEl = document.getElementById('navbar-toggle').children[0];
    navBarToggleIconEl.addEventListener(
      'click',
      () => {
        navBarToggleIconEl.classList.toggle('fa-bars');
        navBarToggleIconEl.classList.toggle('fa-times');
      }
    );
```

Reply:
Fixed